src/index.tsx

```diff
@@ -5,7 +5,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './react_apps/App';
```

Comment:
I think a better name for this folder would be `components` instead of `react_apps`, since each file will probably contain function components that probably won't have `App` in the name - see some of the other branches that have started to modularize, like @lyndseytoft 's `Button` component. This definitely should be it's own folder though (maybe eventually with subfolders, but I imagine `App.tsx` would be at its root)!